apps/penxle.com/src/lib/server/graphql/schemas/post.ts

```diff
@@ -1,3 +1,4 @@
+import { init } from '@paralleldrive/cuid2';
```

Comment:
P3:
```suggestion
import { init as cuid } from '@paralleldrive/cuid2';
```
같은 식으로 cuid 함수명을 명확하게 하는 건 어떤가요?